src/main/kotlin/racingcar/domain/RacingCar.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.domain
+
+class RacingCar(initPosition: Position = Position(0)) {
+
+    var position: Position = initPosition
+        private set
+
+    fun move(movingDirection: MovingDirection) {
+        this.position = this.position.plus(movingDirection.position)
```

Comment:
원시 값인 postion을 포장해 준 부분 매우 좋습니다.

여기서 한번 제안 드릴 부분이 있는데요. 해당 position은 vo 성으로 사용하기 위하여 매번 RacingCar에서 postion을 새로운 Postion으로 변경해 주는 방법으로 사용했을 것이라고 생각됩니다. 하지만 원시 값을 포장해 준 vo를 초기화해줄 필요는 없어 보이며, 오히려 position에 대한 역할도 RacingCar가 모두 가지고 있는 것 같아 보이는 부분이 있습니다.

```suggestion
        this.position.plus(movingDirection.position)
```

위 코드처럼 RacingCar가 postion에게 메시지를 주는 방식으로 변경을 해주는 게 어떨까요? 그리고 value class 특성상 postion class를 새로 교체하는 것보다 값을 더해주는 게 더 좋아 보입니다.